### ISSUE TITLE
[9.1.0] Errors with tags - api uses Snuba, which is not installed

### ISSUE THREAD

**Issue by bp4tsum** [getsentry/sentry#12711]
## Version

Sentry 9.1.0, on-premise (python)

## Description

Hi, got the same error, like in GH-12661,
when trying to filter project Issues by custom tags (browser, os, etc)

Stacktrace for service 'sentry-web':
```
Traceback (most recent call last):
  File "sentry/api/base.py", line 90, in handle_exception
    response = super(Endpoint, self).handle_exception(exc)
  File "sentry/api/base.py", line 190, in dispatch
    response = handler(request, *args, **kwargs)
  File "sentry/api/endpoints/organization_tagkey_values.py", line 34, in get
    query=request.GET.get('query'),
  File "sentry/tagstore/snuba/backend.py", line 609, in get_tag_value_paginator_for_projects
    referrer='tagstore.get_tag_value_paginator_for_projects',
  File "sentry/utils/snuba.py", line 517, in query
    **kwargs
  File "sentry/utils/snuba.py", line 477, in raw_query
    raise SnubaError(err)

SnubaError: HTTPConnectionPool(host='localhost', port=1218): Max retries exceeded with url: /query (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f205472b1d0>: Failed to establish a new connection: [Errno 111] Connection refused',))
```

## Steps to Reproduce
in browser - try to filter project Issues:
Sentry -> Project_name -> Issues -> button "Filter"
```
### HTML:
<a class="btn btn-default toggle-stream-sidebar">
<div class="stream-tag-filter">
<div class="stream-tag-filter">
...
```
Got HTTP 500 (INTERNAL SERVER ERROR) on this request:
```
GET https://sentry.mydomain.com/api/0/organizations/myorg/tags/device/values/?project=9
```

### Note
I applied locally modifications from PR #12683, but nothing happened
(implementation of func 'fetchProjectTagValues')

### What you expected to happen

Get filtered issues

### Possible Solution

Option (in config maybe) to disable Snuba usage

**Comment by markstory** [comment]
When you applied the changes to the javascript did you rebuild static assets with web pack?

**Comment by bp4tsum** [comment]
sorry, no =)
how can I do it? is it enough:
```
yum install nodejs
npm install
npm run build
```

**Comment by markstory** [comment]
I use `node_modules/.bin/webpack` to run webpack. 

> When will this patch be available for sentry upgrade? (with pip)

Not sure, we will likely wait a bit to see if there are any other issues before tagging another release.

**Comment by guidoilbaldo** [comment]
Same problem here, popped up today.
Sentry 9.1 on-premise installation running with docker-compose, sentry-web docker showing error in logs:

```
10.x.x.x - - [06/May/2019:08:46:30 +0000] "GET /api/0/organizations/xxxxx/tags/application_country/values/?project=2 HTTP/1.1" 500 349 "https://sentry.example.com/project/platform/?query=is%3Aunresolved+lastSeen%3A%2B1d" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36"
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/sentry/api/base.py", line 90, in handle_exception
    response = super(Endpoint, self).handle_exception(exc)
  File "/usr/local/lib/python2.7/site-packages/sentry/api/base.py", line 190, in dispatch
    response = handler(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/sentry/api/endpoints/organization_tagkey_values.py", line 34, in get
    query=request.GET.get('query'),
  File "/usr/local/lib/python2.7/site-packages/sentry/tagstore/snuba/backend.py", line 609, in get_tag_value_paginator_for_projects
    referrer='tagstore.get_tag_value_paginator_for_projects',
  File "/usr/local/lib/python2.7/site-packages/sentry/utils/snuba.py", line 517, in query
    **kwargs
  File "/usr/local/lib/python2.7/site-packages/sentry/utils/snuba.py", line 477, in raw_query
    raise SnubaError(err)
SnubaError: HTTPConnectionPool(host='localhost', port=1218): Max retries exceeded with url: /query (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7888cfad90>: Failed to establish a new connection: [Errno 111] Connection refused',))
```

Should I add snuba to my docker-compose? In this case, how?

```
version: '3.5'

x-sentry: &base
  restart: unless-stopped
  image: sentry:9.1.0
  env_file: /data/config/sentry.env
  depends_on:
    - memcached
  links:
    - redis:redis
  volumes:
    - /data/sentry:/var/lib/sentry/files

services:
  sentry:
    <<: *base
    container_name: web
    restart: unless-stopped
    ports:
      - 9000:9000
  worker:
    <<: *base
    container_name: worker
    restart: unless-stopped
    command: run worker
  cron:
    <<: *base
    container_name: cron
    restart: unless-stopped
    command: run cron
  memcached:
    image: memcached:1.5.12
    container_name: memcached
    restart: unless-stopped
  redis:
    image: redis:5.0.3
    container_name: redis
    restart: always
    ports:
      - 6379:6379
```

Would it be sufficient to add something like:

```
  snuba:
    image: getsentry/snuba:latest
    container_name: snuba
    restart: always
    ports:
      - 1218:1218
```

I'm just asking before trying it myself because I would be stopping a production instance and I'd like to be sure it's the right thing to do.
Thanks

**Comment by markstory** [comment]
>   image: sentry:9.1.0

Could you try with `sentry:9.1.1`?

**Comment by guidoilbaldo** [comment]
Yea that's exactly what I did in the end :) 
I'm sorry I wrote the comment above before discovering everything was fixed in 9.1.1 image.
Thanks 👍

**Comment by MPishpecki** [comment]
Same error with sentry, version 10.0.0.dev0 (1699dca5b6e9)
Was someone able to solve this problem?
